src/main/cpp/subsystems/Flywheel.cpp

```diff
@@ -38,7 +38,8 @@ units::radian_t Flywheel::GetAngle() {
 }
 
 units::radians_per_second_t Flywheel::GetAngularVelocity() {
```

Comment:
We should add a comment like this at the top of the function to explain why we're doing this instead of using `GetRate()`:

```cpp
// WPILib uses the time between pulses in GetRate() to calculate velocity, but
// this is very noisy for high-resolution encoders. Instead, we calculate a
// velocity from the change in angle over change in time, which is more precise.
```